Apps/WebClient/src/ClientApp/src/components/private/reports/ImmunizationReportComponent.vue

```diff
@@ -90,13 +90,18 @@ const recommendationFields: ReportField[] = [
         tdAlign: "start",
         thAlign: "start",
         thAttr: { "data-testid": "recommendationTitle" },
```

Comment:
If you're changing data-testid name, isn't the new convention to use kabob  standard i.e., recommendation-title?  Since the name has been changed, were there any functional tests affected?

Reply:
This hasn't changed though